addons/native_dialog/osx_dialog.m

```diff
@@ -4,6 +4,12 @@
 
 #import <Cocoa/Cocoa.h>
 
+/* Normally we should include aintosx.h to get this prototype but it doesn't 
+* work. So repeat the prototype here as a stopgap measure.
```

Comment:
For these block comments, we typically add a space before the '*' from the second line on.